test/specs/photos-timeline.spec.js

```diff
@@ -0,0 +1,25 @@
+/* global describe, it, browser */
+const assert = require('chai').assert
+
+browser.timeouts('script', 60000)
+
+const assertVisualMatch = (result, title) => {
+  assert(result.isWithinMisMatchTolerance, `visual regression for ${title}`)
+}
+
+describe('Photos', function () {
+  it('loads the timeline', function () {
+    browser.url('cozy.local:8080')
```

Comment:
I wonder if it's not on the acceptance testing platform (aka [https://recette.cozycloud.cc](https://recette.cozycloud.cc)) that we should do this kind of testing.